### PR TITLE
Github Action Mypy

### PR DESCRIPTION
**Pull request by Phimaea**

### CODE REVIEW

File: .github/workflows/Mypy.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    name: Mypy
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install Dependencies
+      run: |
+        pip install mypy
```
Comment (sam1penny) [inline .github/workflows/Mypy.yml]: Can you modify this to install from the requirements.txt folder, and add mypy as a dependency?

(If you've installed mypy into a virtual enviroment, run `pip freeze > requirements.txt` in the backend folder to write mypy and the version number)

**Comment by sam1penny** [comment]
Could you also put the `mypy.ini` config file into the backend folder.

**Comment by sam1penny** [comment]
Other than that, looks good to me! (feel free to merge into environment when you've made the changes)

**Comment by MarkHaoxiang** [comment]
Made adjustments on requirements.txt

**Pull request merged**